Sign up form - HTML and Tailwind CSS styling

hallsi90 opened the pull request:
CHANGES IN PR - I've added the css issue to this pull request.

### What does this PR do?

This pull request addresses the signup (HTML) issue and the Sing up (css) issue. I've finalized the Sign Up form including both HTML structure and Tailwind CSS styling. I've also installed Tailwind v4 in the branch, fixing the tailwind issues.

### Description of Task to be completed?

- Created the Sign Up form using semantic HTML
- Styled the form with Tailwind CSS, ensuring responsive and accessible design.
- Resolved Tailwind version issue, allowing proper styling using custom colors and fonts.

### How should this be manually tested?

Clone the repository and navigate to the branch for this pull request : sign-up-ingelinn
Open the HTML file sign-up.html to verify that:
- The page structure follows proper semantic guidelines.
- The sign up form renders correctly and maintains proper styling on different screen sizes.
- The form fields, labels and button are aligned and styled as expected.
- The form is fully responsive, adapting to both mobile and desktop views.
- The Tailwind classes are applied correctly, confirming that the CSS is working as expected.

### Any background context you want to provide?

Previously, there were issues with Tailwind CSS in the project, requiring work in a separate test project and not being able to use the customized styling. Have in mind that I've had to change the tailwind classes after tailwind v4 was fixed in edugate, and be aware of the possibility I have missed some classes that should have been changed.

kristinatonnessen committed:
copied ingelinns code

hallsi90 committed:
Update sign-up.html

hallsi90 committed:
Update sign-up.html

kristinatonnessen reviewed (dismissed):
looks good 😃

myrset reviewed (changes requested):
This file is connected to tailwind v3. Change it to v.4

hallsi90 committed:
Merge branch 'develop' into sign-up-ingelinn

hallsi90 committed:
Updated sign-up-page

Installed tailwind v4.
Made minor changes to the html form.
Changed styling classes to fit custom colors and fonts.
Made form responsive.

hallsi90 commented:
> This file is connected to tailwind v3. Change it to v.4

Done :)

hallsi90 commented:
CHANGES IN PR - I've added the css issue to this pull request -> see description.

myrset reviewed (approved):
Good job on the update!
